pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/ReturnFromFinallyBlock.xml

```diff
@@ -57,4 +57,63 @@ public class Foo {
 }
      ]]></code>
     </test-code>
+    <test-code>
+        <description><![CDATA[
+Return from a Class in Finally
```

Comment:
Although in the other test-code in this file, the description is always given within a CDATA section, I prefer to not use CDATA anymore - since it is easier to read just without CDATA. And it is technically not necessary here.

Reply:
So, I think we will leave this as it is.

Reply:
I have removed add CDATA in the new commit.

Reply:
:+1: